sdk/storage/storage-blob/src/DfsPolicyFactory.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { RequestPolicy, RequestPolicyFactory, RequestPolicyOptions } from "@azure/core-http";
+
+import { DfsPolicy } from "./policies/DfsPolicy";
+
+
+/**
```

Comment:
remove additional line above.

Reply:
Will do

---
In reply to: [318951671](https://github.com/Azure/azure-sdk-for-js/pull/4874#discussion_r318951671) [](ancestors = 318951671)